tests/snapshot-test/helper.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+// @ts-ignore
+const { loadESLint } = require('eslint');
```

Comment:
`loadESLint` is preferred to support both `eslintrc` and Flat Config configuration files. This function is available since ESLint v8.57.0, so I've updated our development ESLint.

[ESLint v8.57.0 released - ESLint - Pluggable JavaScript Linter](https://eslint.org/blog/2024/02/eslint-v8.57.0-released/#new-api-loadeslint())